app/boards/shields/zmk_uno/zmk_uno_split_right.overlay

```diff
@@ -11,7 +11,7 @@
 };
 
 &split_direct_matrix_transform {
-    row-offset = <2>;
+    col-offset = <4>;
```

Comment:
```suggestion
    row-offset = <1>;
```
If I am not mistaken, this seems to be the right value

Reply:
Fixed. Thanks!

Reply:
I think there might have been a mistake, you converted `col-offset` value to `row-offset` but the value is still `<4>`, not `<1>`.

Reply:
Indeed. That's what I get for trying to quickly fix things during a rebase. Fixed?

Reply:
LGTM now 😄